docs/create-image-and-azure-resources.md

```diff
@@ -11,8 +11,8 @@ After successful image generation, a snapshot of the temporary VM will be conver
 - `OS` - Windows/Linux
 - `packer` - Can be downloaded from https://www.packer.io/downloads
 - `PowerShell 5.0 or higher` or `PSCore` for linux distributes.
-- `Azure CLI ` - https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
-- `Azure Powershell module` - https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-4.6.1
+- `Azure CLI ` - https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
```

Comment:
Do we need azure-cli for some tasks or it can be removed from the list?

Reply:
Yep, we use it in the https://github.com/actions/virtual-environments/blob/main/helpers/CreateAzureVMFromPackerTemplate.ps1 script.